src/backend/api/Fusion.Resources.Api/Controllers/Departments/DepartmentsController.cs

```diff
@@ -85,31 +85,40 @@ public async Task<ActionResult> AddDelegatedResourceOwner(string departmentStrin
         public async Task<ActionResult<List<ApiDepartment>>> GetPositionDepartments(
             Guid projectId, Guid positionId, Guid instanceId)
         {
-            var result = new List<ApiDepartment>();
+            var relevantDepartments = new List<ApiDepartment>();
+
             var position = await orgApiClient.GetPositionV2Async(projectId, positionId);
             if (position is null) return NotFound();
 
-            var department = await DispatchAsync(new GetDepartment(position.BasePosition.Department));
-            if(department is not null) result.Add(new ApiDepartment(department));
+            QueryDepartment? department = null;
+            QueryRelevantDepartments? related = null;
 
-            var command = new GetRelevantDepartments(position.BasePosition.Department);
-            var relevantDepartments = await DispatchAsync(command);
+            // Empty string is a valid department in line org (CEO), but we don't want to return that.
+            if (!string.IsNullOrWhiteSpace(position.BasePosition.Department))
+            {
+                department = await DispatchAsync(new GetDepartment(position.BasePosition.Department));
+                related = await DispatchAsync(new GetRelevantDepartments(position.BasePosition.Department));
+            }
 
             // TODO: lookup based on responsibility matrix
```

Comment:
Is the TODO something we want to do?

Reply:
Yes, but it's not completely clear how that should work. I think there is a devops work item for it, if not I'll add one.